pkg/tracer/tracer.go

```diff
@@ -162,8 +163,15 @@ func NewFlowFetcher(cfg *FlowFetcherConfig) (*FlowFetcher, error) {
 		}
 
 		enableFlowFiltering := 0
+		hasFilterSampling := uint8(0)
 		if cfg.EnableFlowFilter {
 			enableFlowFiltering = 1
+			for _, f := range cfg.FilterConfig {
```

Comment:
nit: we can have this as function in flow_filter.go and call it or even move the flow_filter processing up and let it return the flag not urgent can be done later